rust/main/hyperlane-core/src/types/reorg.rs

```diff
@@ -0,0 +1,20 @@
+use derive_new::new;
+use serde::{Deserialize, Serialize};
+
+use crate::H256;
+
+/// Details about a detected chain reorg, from an agent's perspective
+#[derive(Debug, Clone, Serialize, Deserialize, new)]
+pub struct ReorgEvent {
+    /// the merkle root built from this agent's indexed events
+    pub local_merkle_root: H256,
+    /// the onchain merkle root
+    pub canonical_merkle_root: H256,
+    /// the index of the checkpoint when the reorg was detected
+    /// (due to a mismatch between local and canonical merkle roots)
+    pub checkpoint_index: u32,
+    /// the timestamp when the reorg was detected, in seconds since the Unix epoch
+    pub unix_timestamp: u64,
+    /// (optional) the height of the block when the reorg was detected
+    pub reorg_period: u64,
```

Comment:
iiuc the doc for this is different from how it's used - here it says it's the block height but I think it's passed the configured reorg period

Reply:
oops, I couldn't get the block height but though it'd be useful to post the configured reorg period just in case. The timestamp should be a reasonable indication for the block height

Reply:
will fix the doc comments